exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/FileSystemPlugin.java

```diff
@@ -57,7 +61,9 @@
  */
 public class FileSystemPlugin extends AbstractStoragePlugin {
 
-  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(FileSystemPlugin.class);
+  private static final Logger logger = LoggerFactory.getLogger(FileSystemPlugin.class);
+
+  private static final List<String> BUILT_IN_CODECS = Collections.singletonList(ZipCodec.class.getCanonicalName());
```

Comment:
Are no other codecs provided "out of the box"? For others, I need to provide a jar and set a config option? Or, should we move the other built-in ones here and out of the config file?

Reply:
`org.apache.hadoop.io.compress` library supports gzip / bzip2 out of box. Here we only need to add codecs that are missing in this library. I have updated parameter name and added comment to avoid the confusion. `TestCompressedFiles` contains tests for all supported formats.